examples/pyomobook/strip_examples.py

```diff
@@ -0,0 +1,28 @@
+import glob
+import sys
+import os
+import os.path
+
+def f(file):
+    base, name = os.path.split(file)
+    prefix = os.path.splitext(name)[0]
+    if prefix.endswith('_strip'):
+        return
+    OUTPUT = open(base+'/'+prefix+'_strip.py','w')
+    INPUT = open(file,'r')
+    for line in INPUT:
+        if line[0] == '#' and '@' in line:
+            continue
+        OUTPUT.write(line)
+    INPUT.close()
+    OUTPUT.close()
+    #if not os.path.exists(base+'/'+prefix+'.txt'):
```

Comment:
Are lines 19/20 necessary?

Reply:
I'll admit I didn't touch or even look at this file. It's used as part of the custom build process for the book text. I'll take a quick look at it and make the minor changes you suggested.